modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java

```diff
@@ -1749,16 +1760,41 @@ private Map<String, String> _addOrUpdateDDMStructures(
 		return ddmStructuresIdsStringUtilReplaceValues;
 	}
 
-	private void _addOrUpdateDDMTemplates(
+	private Map<String, String> _addOrUpdateDDMTemplates(
 			DDMStructureLocalService ddmStructureLocalService,
 			ServiceContext serviceContext)
 		throws Exception {
 
+		Map<String, String> ddmTemplateIdsStringUtilReplaceValues =
```

Comment:
this map does not contains just ddm templates anymore
please rename to following the existing pattern like: ddmTemplateIdsAndTemplateEntryIdsStringUtilReplaceValues